src/routes/room.route.js

```diff
@@ -0,0 +1,9 @@
+const router = require('express').Router()
+const roomController = require('../controllers/room.controller')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Backend/pullRequest?prid=7180713)
```suggestion
const roomController = require("../controllers/room.controller")
```